### PR TITLE
Fixes #1109: Add feature to report skills

### LINKED ISSUE #1109: Add feature to report skills in Skill Details section
**Actual Behaviour**
Skill reporting feature absent.

**Expected Behaviour**
It should be present to allow users to report skills.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by anshumanv**
Fixes #1109 

Changes: Add feature to report skills.

Surge Deployment Link: https://pr-1233-fossasia-susi-skill-cms.surge.sh

Screenshots for the change:
image

### CODE REVIEW

**Comment by anshumanv** [comment]
Snackbar :+1: 
I made it highlight Cancel by default so user doesn't accidentally submit it if he presses enter by mistake.

**Comment by fragm3** [comment]
@anshumanv change its style please, make it like this:
screen shot 2018-07-21 at 2 19 41 pm

**Comment by anshumanv** [comment]
Updated, dropped the default focus.

File: src/components/SkillPage/SkillListing.js [hunk 1]
```diff
@@ -594,6 +603,50 @@ class SkillListing extends Component {
...
+      showReportDialog: !this.state.showReportDialog,
+    });
+  };
+
+  handleReportSubmit = () => {
+    let reportUrl = `${urls.API_URL}/cms/reportSkill.json?model=${
+      this.state.skillModel
+    }&group=${this.state.skillGroup}&language=${
+      this.state.skillLanguage
+    }&skill=${this.state.skill_name}&feedback=${
```
Comment (akshatnitd) [inline src/components/SkillPage/SkillListing.js]: Use `skill_tag` of the skill metadata object for skill name. Due to which, it gives error, when I report the skill twice.

Comment (anshumanv) [reply src/components/SkillPage/SkillListing.js]: Still throws an error when skill tag is used.

**Comment by anshumanv** [comment]
Updated the color!
~skill_name~ skill_tag

**Comment by anshumanv** [comment]
@AnupKumarPanwar @DNS-404 @hkedia321 please review.

**Pull request merged**